src/System.Configuration.ConfigurationManager/tests/System.Configuration.ConfigurationManager.Tests.csproj

```diff
@@ -92,9 +92,13 @@
     <Compile Include="System\Configuration\StringUtilTests.cs" />
     <Compile Include="System\Configuration\TempConfig.cs" />
     <Compile Include="System\Configuration\TestData.cs" />
+    <Compile Include="System\Configuration\TimeSpanValidatorAttributeTests.cs" />
     <Compile Include="System\Configuration\TypeUtilTests.cs" />
     <Compile Include="System\Configuration\UriSectionTests.cs" />
     <Compile Include="System\Configuration\ValidatiorUtilsTests.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL" />
```

Comment:
Does it work without this? If I look at the other test projects, we don't do this. I think we auto include references. Although I do see it in `System.IO.Ports.Tests.csproj` it probably shoudn't be there.